lib/rabblebot/plugins/weather.rb

```diff
@@ -0,0 +1,86 @@
+require 'open-uri'
+require 'json'
+
+class RabbleBot
+  attr_reader :google_api, :forecastio_api
+  
+  module RabbleBotPlugin
+    # a weather plugin
+    class Weather < BasicPlugin
+      def initialize(bot)
+        super(bot)
+        @bot.info 'Loading Weather Plugin'
+        @google_api = 'YOUR_GOOGLEAPI_KEY'
```

Comment:
once you've added the config, you can say, `@google_api = @config['google_api']`